spec/index.html

```diff
@@ -2884,6 +2884,17 @@ <h3>Dataset Poisoning</h3>
     </p>
   </section>
 
+  <section id="insecure-hashing-algorithms" class="informative">
+    <h3>Insecure Hashing Algorithms</h3>
+
+    <p>It is possible that the hashing algorithm used by RDFC-1.0 may become
```

Comment:
Referencing the definition of the hash algorithm would be useful, as it also makes this statement. (Alternatively, use the reference as "hash algorithm" directly, instead of "hashing algorithm").

```suggestion
    <p>It is possible that the <a data-lt="hash algorithm">hashing algorithm</a> used by RDFC-1.0 may become
```

Reply:
+1 for "hash algorithm", better than "hashing algorithm"

Reply:
> +1 for "hash algorithm", better than "hashing algorithm"

I updated the suggestion accordingly.

Reply:
"may" => "might" to avoid normative language.